Shared_AIHS2/Skin/Hooks.cs

```diff
@@ -40,12 +40,12 @@ private static void OverlayBlit(Texture source, RenderTexture dest, Material mat
             var controller = instance.trfParent?.GetComponent<KoiSkinOverlayController>();
             if (controller != null)
             {
-                if (controller.ChaControl.customTexCtrlFace?.createCustomTex?.Contains(instance) == true)
+                if (controller.ChaControl.customTexCtrlFace?.createCustomTex?.FirstorDefault() == instance)
```

Comment:
If possible, it would be much better to use some property that points specifically to the albedo CustomTextureCreate, so that there is no chance for errors.
 
Is albedo always guaranteed to be the first texture? If yes, then it would be better to access it by index `[0]` and add a comment explaining the reason for it.

Reply:
There's no property or similar, sadly; in the game, it's accessed specifically by index in the array. But I can confirm that from what I'm seeing, the array is always initialized alongside the parent `CustomTextureControl`, so as long as the `CustomTextureControl` is not null, it should always have a valid `CustomTextureCreate` array and the albedo will always be at index 0. With that in mind, I think it should be safe to use `customTexCtrlFace?.createCustomTex[0]` and `customTexCtrlBody?.createCustomTex[0]`, respectively.